node/nodeRunner.go

```diff
@@ -77,7 +77,7 @@ const (
 type nodeRunner struct {
 	configs *config.Configs
 
-	createManagedProcessComponentsMethod   func(processArgs processComp.ProcessComponentsFactoryArgs) (mainFactory.ProcessComponentsHandler, error)
+	createManagedProcessComponentsMethod func(processArgs processComp.ProcessComponentsFactoryArgs) (mainFactory.ProcessComponentsHandler, error)
```

Comment:
I think this could be removed and also we could rolled back all the changes in this file

Reply:
right, will remove

Reply:
done